internal/kafka/internal/services/kafka.go

```diff
@@ -960,52 +962,107 @@ type KafkaStatusCount struct {
 	Count  int
 }
 
-type KafkaRegionCount struct {
+type KafkaStreamingUnitCountPerRegion struct {
 	Region        string
-	InstanceType  string `gorm:"column:instance_type"`
-	ClusterId     string `gorm:"column:cluster_id"`
+	InstanceType  string
+	ClusterId     string
 	Count         float64
-	CloudProvider string `gorm:"column:cloud_provider"`
+	CloudProvider string
 }
 
-func (k *kafkaService) CountByRegionAndInstanceType() ([]KafkaRegionCount, error) {
+func (kafkaStreamingUnitCountPerRegion KafkaStreamingUnitCountPerRegion) isSame(kafkaPerRegionFromDB *KafkaPerRegionCount) bool {
```

Comment:
maybe `isSameInstanceType` would make more sene here?

Reply:
It checks for several things, instance type being one of them.